### ISSUE TITLE
Content offset doesn't work

### ISSUE THREAD

**Issue by ghost** [bannzai/SpreadsheetView#89]
Hi, I'm creating a spreadsheet view that gets its data periodically refreshed. Whenever I reload data, I want to remember where the user scrolled to. However, the view will always reload to the beginning of the scroll view. Here's my code:

```
override func willMove(toSuperview newSuperview: UIView?) {
        super.willMove(toSuperview: newSuperview)
        timer = Timer.scheduledTimer(timeInterval: 2.0, target: self, selector: #selector(updateContents), userInfo: nil, repeats: true)
        timer.fire()
    }


func updateContents() {
        let contentOffset = spreadsheetView.contentOffset
        print(contentOffset)
        spreadsheetView.reloadData()
        spreadsheetView.contentOffset = contentOffset
}
```
I'm printing out the contentOffset values and they seem to be calculated correctly.
<img width="150" alt="screen shot 2017-06-30 at 11 16 25 am" src="https://user-images.githubusercontent.com/12378366/27748805-9cf8033c-5d85-11e7-9979-f2ccf9e3a736.png">

Do you know what the issue might be? Thanks!

**Comment by kishikawakatsumi** [comment]
@zzhao36 Spreadsheet view keeps `contentOffset` after reloading. It is the same behavior as UITableView or UICollectionView. I'd like to know what happens correctly. Can you provide a reproducible sample code?

**Comment by kishikawakatsumi** [comment]
Closing due to no response

**Comment by ghost** [comment]
Hi, sorry for the super late reply! Essentially, I was constantly reloading my spreadsheet view to update the information in the cells. Basically, the content offset would always be off:
![aug-04-2017 11-03-50](https://user-images.githubusercontent.com/12378366/28981028-d0f4f826-7904-11e7-8cef-b686713bb726.gif)
I think it's because reloadData() was called before the scrolling stopped, and therefore the content offset was not the actual, intended content offset. I fixed this by adding a new variable 'isScrolling' to the spreadsheet view, and then modified the scrollview delegate:

```
extension SpreadsheetView: UIScrollViewDelegate {
       public func scrollViewDidScroll(_ scrollView: UIScrollView) {
               isScrolling = true
               // more code
       }

       public func scrollViewDidEndScrollingAnimation(_ scrollView: UIScrollView) {
               isScrolling = false
               // more code
       }
}
```

And then when I called my reload function, I checked to make sure the user wasn't scrolling:

```
// fires once a second!
func shouldUpdateSpreadsheetView() {
        if isScrolling { return }
        // more code
}
```

In the future, I think there should be two optional functions for SpreadsheetViewDelegate exposed in the public API that does something similar to what I did here - `spreadsheetViewDidScroll`, and `spreadsheetViewDidEndScrollingAnimation`. Please let me know if you have any questions, thanks!

**Comment by kishikawakatsumi** [comment]
@zzhao36 It probably has been fixed by https://github.com/kishikawakatsumi/SpreadsheetView/pull/117. Can you check if the problem is fixed in `master` branch or `v0.8.1`? (I will release `v0.8.1` very soon.)

**Comment by kishikawakatsumi** [comment]
I believe this was fixed by https://github.com/kishikawakatsumi/SpreadsheetView/pull/117. If you still in trouble, feel free reopen this issue or create a new issue. Thanks for reporting!
